W-15818513-dw-functions-27-duke (#323)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-mime-functions-ishandledby.adoc

= isHandledBy
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]

[[ishandledby1]]
== isHandledBy&#40;base: MimeType, other: MimeType&#41;: Boolean

> Returns `true` if the given `MimeType` value can be handled the base `MimeType` value.

Reviewer: ```suggestion
Returns `true` if the given `MimeType` value is handled by the base `MimeType` value.
```

_Introduced in DataWeave version 2.7.0._

=== Parameters

[%header, cols="1,1,3"]
|===
| Name | Type | Description
| `base` | `MimeType` | The MIME type value used as baseline.
| `other` | `MimeType` | The MIME type value to be validated.
|===

=== Example

> This example test several MIME types handled validation.

Reviewer: ```suggestion
This example tests how MIME types handles several validations.
```
